src/m365/commands/logout.ts

```diff
@@ -41,7 +109,32 @@ class LogoutCommand extends Command {
     }
 
     this.initAction(args, logger);
-    await this.commandAction(logger);
+    await this.commandAction(logger, args);
+  }
+
+  private async getIdentityToLogout(options: Options): Promise<Identity | undefined> {
+    try {
+      if (!options.identityId && !options.identityName) {
+        return;
+      }
+
+      const identities = auth.service.availableIdentities!.filter(i => i.identityName === options.identityName || i.identityId === options.identityId);
+
+      if (identities.length === 0) {
+        throw new Error(`The identity '${options.identityId || options.identityName}' cannot be found.`);
+      }
+
+      if (identities.length > 1) {
+        const resultAsKeyValuePair = formatting.convertArrayToHashTable('identityId', identities);
```

Comment:
That would mean a change in the central disambiguation prompt code. I agree that is a good thing, but let's pick that up in a separate issue.

Reply:
What if we introduced a new property that contains the combined values or would that still not work?

Reply:
That is possible yes. but I think I would exclude the name. That value could otherwise become quite long and therefore force a broad terminal window. 

When I work with application identities I tend to roughly know what objectID's I'm working with, but I agree it would be nicer to have the tenant ID as well. It would be even better to have the tenant name, but I'm not sure we have that available.

Reply:
We don't have it readily available but we even have a command for it! `tenant info get` 😄

Reply:
We did not discuss this further by the way, should we just leave it?